python/erdos/message.py

```diff
@@ -1,11 +1,13 @@
 import pickle
-from typing import Any
+from typing import Any, Generic, TypeVar
```

Comment:
```suggestion
from typing import Generic, TypeVar
```

`Any` is not being used anymore. Please remove.